packages/cli/cloud/src/utils/tests/compress-files.test.ts

```diff
@@ -0,0 +1,37 @@
+import path from 'path';
+import os from 'os';
+import { isIgnoredFile } from '../compress-files';
+
+describe('isIgnoredFile', () => {
+  const folderPath = os.tmpdir(); // Use the system's temporary directory as a dummy folder path
```

Comment:
You could use [mock-fs](https://www.npmjs.com/package/mock-fs) here to avoid real effects in the test runs

Reply:
fyi we use memfs already

Reply:
Oh yeah, that one, I misremembered which one we used 😆

Reply:
ping on this before merge

Reply:
I just pinged on unresolved issues with no answers :) You can resolve if it's not necessary to do anything ;)